src/sndfile.c

```diff
@@ -8,7 +8,7 @@
 **
 ** This program is distributed in the hope that it will be useful,
 ** but WITHOUT ANY WARRANTY; without even the implied warranty of
-** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the
```

Comment:
??

Reply:
Artifact of using tabify in emacs; picked a space which would be the same as tab.  Fixed this and the others I believe.